### DISCUSSION TITLE
比较疑惑，为什么会给出很多网上论坛内容

### DISCUSSION THREAD

**Discussion by mshmoon** [ymcui/Chinese-LLaMA-Alpaca#840] [Q&A]
请问，你知道化学专业就业前景吗？
答：知道，不过不怎么喜欢。因为太枯燥了！ 而且我还没毕业呢~ 我现在是学的计算机专业的哦~~ 所以我可以推荐你去试试看吧! 你会发现这个方向发展得挺快啊~~~ 如果你想要了解更多的信息请跟我联系喔～谢谢啦!!! ## 请把你所做的事情写下来并写上你的想法和建议（不要超过100字）．然后把答案发送到我们的邮箱 info@edudogzhuanqiyejiaoyu .com 我们将根据您的情况为您提供帮助与指导...我们期待您成为一名优秀的教育工作者.... #include{int main() {printf("please input your age\n");scanf("%d",&age);if(isokay)return;else printf("\nthe answer to this question depends on you!\nmust be between 1 and 100.\rprintaquestion();} } ```````` 如何使用 1. 在本课程中完成所有作业。2. 每周学习一个新单元。3. 通过阅读理解课文内容、练习巩固知识点及技能；4. 根据老师布置的任务做笔记。1. 本节共5个小题 2. 第一大项为“填空” 3. 第二章第1页 3-4段 4. 第三部分 4-5 小问 5. 第4分 5-6句 6. 最后一则 7-8句话 7.最后8-9句子 8-10词 9-11语气 11-12单词 12-13词语 13-14一句 14-15两句 15-16一段 16-17短诗 17-18片断 18-19笔划 20-22行 23-24条23-21步25-37处26-30次303132时343536期373839款405060篇415162道425263部435464例444565人465666份475767地485968号4960070件49.61007101名20212213天291027日3003303月400380小时5005305分钟58066秒860775元890890万元940991位9510102%9911123%1051106%961208%981302%971404%99.15145%10.16179%11.18287%12.193330%13.22030%14.24070%15.25080%16.280100%17.3201%18.20020%19.202095%21.21192%)22.19093%.23.20004%。24.40003.25.0000045.26.

**Comment by ymcui** [comment]
用的什么模型？正常Alpaca模型不会是这种输出。example目录下有示例输出，你应该先debug一下是否能得出与示例类似的答案。
